docs/server/introduction.mdx

```diff
@@ -3,22 +3,55 @@ title: Server SDKs
 slug: /server/introduction
 ---
 
-Statsig Server SDKs evaluate gate, config, experiment checks locally, on your server.
+Statsig server SDKs enable you to run experiments for any applications improvements driven by server-side functionality including APIs, algorithms, latency, configuration, and infrastructure improvements. Statsig server SDKs evaluate gate, config, and experiment checks locally on your application server.
+  
+As shown in the diagram below, implementing an experiment using a Statsig server SDK consists of three phases:
+1. Initialization
+2. Evaluating Variants
+3. Recording Exposure Checks & Custom Events
 
-In general, Server SDKs follow the same pattern:
+![Server SDKs](https://user-images.githubusercontent.com/1315028/169563936-121cb75e-37e5-491f-8bb1-2d853c2e04b8.png)
 
-1. Initialize the SDK.  The SDK will reach out to Statsig for the most up to date definition of all of your feature gates, dynamic configs, experiments, autotunes, and layers
-2. Evaluate `checkGate`/ `getConfig` / `getExperiment` calls locally
-3. Batch exposure logs with calls to `logEvent` and periodically flush logs to Statsig servers
+### 1. Initialization
+-	The server SDK's `initialize` call takes the **server SDK key** and makes a network request to Statsig servers; this network call fetches all rule sets to your application server to enable local evaluation in the next phase
```

Comment:
rule sets is a weird term  for newbies.  Id just say "fetches the definition of all configurations in your statsig project"